src/pages/installation.md

```diff
@@ -27,9 +27,9 @@ Node.js ships their own package manager during installation, so you would have *
     - [Yarn](https://yarnpkg.com/getting-started)
     - [pnpm](https://pnpm.io/installation)
 
-We ask that your table and chairs and in upright postilions, and you remain seated until after we take-off.
+We ask that your table and chairs are in upright postilions, and you remain seated until after we take-off.
```

Comment:
I think there is a typo in `postilions`, or it is a space term I don't know xD

Reply:
dear god, my dyslexia finds new ways of even evading spell checkers 😣Ill sort that out asap

Reply:
oh no xD you might find the Grammarly plugin for VS Code useful then, it's saved me a couple of times!

Reply:
There's a grammerly plugin!!  that thing saved my ass in 4th yr, honestly my super thought I took extra english tuition classes in the summer lol